### PR TITLE
Merge 1.4 1.5

### PR DESCRIPTION
**Pull request by philippemossiere**

### CODE REVIEW

**Comment by willy-ahva** [comment]
GTM

**Comment by jjanvier** [comment]
gtm

**Comment by solivier** [comment]
gtm

File: features/Context/DataGridContext.php [hunk 1]
```diff
@@ -859,8 +859,11 @@ public function iResetTheGrid()
      */
     public function iRefrestTheGrid()
     {
-        $this->datagrid->clickOnRefreshButton();
-        $this->wait();
+        $this->spin(function () {
+            $this->getSession()->getPage()->clickLink('Refresh');
```
Comment (aRn0D) [inline features/Context/DataGridContext.php]: Not sure about that, normally a "spin" will execute the callback every second?

**Pull request merged**